Document Google Cloud CLI access (#19905)
gravitational/teleport: docs/pages/reference/cli.mdx

$ tsh apps ls
```

### tsh gcloud

Proxy `gcloud` CLI commands through the Teleport Application Service. `gcloud`
is a tool for interacting with Google Cloud. A user must already be
authenticated to a Google Cloud application in Teleport before they can execute
`tsh gcloud` commands.

```code
$ tsh gcloud [--app] [<command>] 
```

#### Arguments

`command`: A `gcloud` command to run, including arguments and flags.

#### Flags

| Name | Default Value(s) | Allowed Value(s) | Description |
| - | - | - | - |

> | `--app` | Currently logged in Google Cloud application | The name of a Google Cloud application as listed via `tsh apps ls`. | The Google Cloud application to run the command against, if logged in to multiple Google Cloud applications. |

Reviewer: Employs `<br/>` tags to prevent `--app` from being broken across lines, until https://github.com/gravitational/docs/issues/126 is resolved.

```suggestion
| `--app` | Currently logged in<br/>Google Cloud application | The name of a Google Cloud<br/>application as listed via `tsh apps ls`. | The Google Cloud application to run<br/>the command against, if logged in<br/> to multiple Google Cloud applications. |
```

#### Global flags

These flags are available for all commands `--login, --proxy, --user, --ttl, --identity, --cert-format, --insecure, --auth, --skip-version-check, --debug, --jumphost`.
Run `tsh help <subcommand>` or see the [Global Flags section](#tsh-global-flags).

#### Examples

```code
$ tsh gcloud compute instances list 
```

### tsh gsutil

Proxy `gsutil` CLI commands through the Teleport Application Service. `gsutil`
is a tool for interacting with Google Cloud Storage. A user must already be
authenticated to a Google Cloud application in Teleport before they can execute
`tsh gsutil` commands.

```code
$ tsh gsutil [--app] [<command>] 
```

#### Arguments

`command`: A `gsutil` command to run, including arguments and flags.

#### Flags

| Name | Default Value(s) | Allowed Value(s) | Description |
| - | - | - | - |

> | `--app` | Currently logged in Google Cloud application | The name of a Google Cloud application as listed via `tsh apps ls`. | The Google Cloud application to run the command against, if logged in to multiple Google Cloud applications. |

Reviewer: Same as above:

```suggestion
| `--app` | Currently logged in<br/>Google Cloud application | The name of a Google Cloud<br/> application as listed via `tsh apps ls`. | The Google Cloud application to run<br/>the command against, if logged in to multiple Google Cloud applications. |
```

$ aws s3 ls
```

### tsh proxy gcloud

Start a local proxy for Google Cloud API access. The user must already be logged
in to at least one Google Cloud application via Teleport before the proxy can
start.

```code
$ tsh proxy gcloud [<flags>] 
```

#### Flags

> | Name | Default Value(s) | Allowed Value(s) | Description |

Reviewer: This table needs formatting as well, but I wasn't able to find the magic `<br/>` placement to fix it.

| - | - | - | - |
| `--app` | Currently logged in Google Cloud app | string | Optional name of the Google Cloud application (as shown in `tsh apps ls`) to use if logged in to multiple |
| `-p, --port` | none | port number | Specify the source port for the local proxy |

> | ` -e, --endpoint-url` | HTTP Proxy | Endpoint URL | Run the local proxy to serve as an AWS endpoint URL. If not specified, the local proxy serves as an HTTPS proxy. |

Reviewer: I'm guessing:

```suggestion
| ` -e, --endpoint-url` | HTTP Proxy | Endpoint URL | Run the local proxy to serve as a Google Cloud endpoint URL. If not specified, the local proxy serves as an HTTPS proxy. |
```
